scripts/utils.js

```diff
@@ -213,11 +218,6 @@ async function incrementRemixVersion(nextVersion) {
 
   // Update deployment script `@remix-run/dev` version
   await updateDeploymentScriptVersion(nextVersion);
-
-  // Commit and tag
-  execSync(`git commit --all --message="Version ${nextVersion}"`);
```

Comment:
Why are we removing this?

Reply:
We're using Changeset for creating versions, not a manual commit

Reply:
we still manually commit for nightly and experimental releases